pytket/pytket/circuit/display/__init__.py

```diff
@@ -66,43 +68,95 @@ def _render(self, filename: str) -> markupsafe.Markup:
 jinja_env = Environment(loader=loader, extensions=[IncludeRawExtension])
 
 RenderCircuit = Union[Dict[str, Union[str, float, dict]], Circuit]
+Orientation = Literal["row"] | Literal["column"]
+
+
+@dataclass(kw_only=True)
+class RenderOptions:
```

Comment:
Is this class intended to be user facing? Or does it just internally store the values passed to `save_render_options()`? I think the latter.